src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml.cs

```diff
@@ -132,16 +133,24 @@ async void InitializeAsync()
             VirtualFolderPath = string.Empty;
             try
             {
-                if (viewModel.Link != null && !string.IsNullOrEmpty(viewModel.CurrentPackageName))
+                //if this node is from a package then we set the virtual host path to the packages docs directory.
+                if (viewModel.Link != null && !string.IsNullOrEmpty(viewModel.CurrentPackageName) && viewModel.IsOwnedByPackage)
                 {
-                    var absolutePath = Path.GetDirectoryName(HttpUtility.UrlDecode(viewModel.Link.AbsolutePath));
-                    //We move two levels up so it will be located in same level than the the NodeHelpSharedDocs directory
-                    var imagesLocation = new DirectoryInfo(absolutePath).Parent.Parent.FullName;
-                    //Adds the NodeHelpSharedDocs directory to the path
-                    VirtualFolderPath = Path.Combine(imagesLocation, SharedDocsDirectoryName);
+                    VirtualFolderPath = Path.GetDirectoryName(HttpUtility.UrlDecode(viewModel.Link.AbsolutePath));
                 }
+                //if the node is not from a package, then set the virtual host path to the shared docs folder.
+                else if (viewModel.Link != null && !viewModel.IsOwnedByPackage)
+                {
+                    VirtualFolderPath = Path.Combine(new FileInfo(Assembly.GetExecutingAssembly().Location).DirectoryName, SharedDocsDirectoryName);
+                }
+                //unclear what would cause this.
                 else
+                {
                     VirtualFolderPath = FallbackDirectoryName;
+                }
+                //TODO - the above will not handle the case that a package's images/dyns are located in the shared folder
```

Comment:
I'm not sure anyone will want to do this, but I can imagine them asking, I think a better solution is to allow sharing the images and dyns directly out of the main `/doc` folder of each package though - which I think the current code will do.